src/commands/curl.js

```diff
@@ -15,16 +16,46 @@ async function loadTokens () {
   };
 }
 
+function printHelp () {
+  const apiDocUrlv2 = 'https://developers.clever-cloud.com/api/v2/';
+  const apiDocUrlv4 = 'https://developers.clever-cloud.com/api/v4/';
+
+  println(`Usage: clever curl
+Query Clever Cloud's API using Clever Tools credentials. For example: 
+
+  clever curl ${conf.API_HOST}/v2/self
+  clever curl ${conf.API_HOST}/v2/summary
+  clever curl ${conf.API_HOST}/v4/products/zones
+  clever curl ${conf.API_HOST}/v2/organisations/<ORGANISATION_ID>/applications | jq '.[].id'
+  clever curl ${conf.API_HOST}/v4/billing/organisations/<ORGANISATION_ID>/<INVOICE_NUMBER>.pdf > invoice.pdf
+
+Our API documentation is available here : 
+
+  ${apiDocUrlv2}
+  ${apiDocUrlv4}`);
+}
+
 async function curl () {
 
+  const curlNeedsHelp = process.argv[2] !== 'curl' || process.argv.length < 3 || process.argv[3] === '--help' || process.argv[3] === '-h';
```

Comment:
I don't know if it's easy but in that case the --help requires to be set in 3rd position. Shouldn't we plot the help too if customer enters: 

```sh
curl https://api.clever-cloud.com/v2/self --help
```

Reply:
I think the condition is already complex enough, and user can type:

```
clever curl --help https://api.clever-cloud.com/v2/self
```

If we add this condition, it will not work on many case if there is a cURL argument for exemple. And it will be without an end. We control everything before the `curl` we shouldn't try to handle what's coming after